packages/devtools_app/test/flutter/timeline_screen_test.dart

```diff
@@ -23,8 +23,44 @@ import 'wrappers.dart';
 
 void main() {
   TimelineScreen screen;
+  TimelineScreenBodyState state;
   FakeServiceManager fakeServiceManager;
 
+  Future<void> buildTimelineScreen(
+    WidgetTester tester,
+    TimelineMode mode, {
+    TimelineController controller,
+  }) async {
+    // Set a wide enough screen width that we do not run into overflow.
+    await setWindowSize(const Size(1599.0, 1000.0));
```

Comment:
call resetWindowSize() at the end of each test that uses this to avoid changing the window size for other tests.